src/module/types/template/Matrix.ts

```diff
@@ -52,6 +52,7 @@ export const MatrixData = () => ({
     link_locked: new BooleanField(),
     hot_sim: new BooleanField(),
     running_silent: new BooleanField(),
+    public_grid_modifier: new NumberField({ required: true, nullable: false, integer: true, initial: -2 }),
```

Comment:
This literal value should be part of the same Rules file as the other one. That way we can be sure to not have these values throughout the system and changes to it only needing one place.